### ISSUE TITLE
editor.unicodeHighlight.ambiguousCharacters not accessible

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#6326]
Have an ambiguous character in your editor. Notice how we render this specially so user can detect the ambiguous character.
Turn on a screen reader and notice that the screen reader does not ready anything specially when you visit the line with the ambiguous character. 

Some ideas on what we could do:
- play an aria live event when caret is on the ambiguous character. The aria live event would announce the exact character code.
- play an audio cue when on a line with an ambiguous character

Assigning to September so we reserve some time to think about this next milestone.
